src/main/kotlin/com/exactpro/th2/kafka/client/Main.kt

```diff
@@ -124,7 +125,20 @@ fun main(args: Array<String>) {
     LOGGER.info { "Microservice shutted down." }
 }
 
-class EventSender(private val eventRouter: MessageRouter<EventBatch>, private val rootEventId: EventID) {
+class EventSender(private val eventRouter: MessageRouter<EventBatch>, private val rootEventId: EventID) : AutoCloseable {
+
+    private val batchSenderExecutor = Executors.newSingleThreadScheduledExecutor()
+    private val eventBatcher = EventBatcher(
+        maxBatchSizeInBytes = 512 * 1024,
```

Comment:
Should be taken from cradle configuration

Reply:
getCradleConfiguration is depricated, getCradleNonConfidentialConfiguration() is 'protected'. Probably we should add these params to custom config.